README.md

```diff
@@ -80,9 +85,9 @@ SOLR_URL=http://localhost:8983/solr bundle exec cap development "collections:del
 
 ## SolrCloud Backups
 
-Backups are implemented as a ruby service class wrapped in a rake task that's invoked by cron (scheduled via whenever / capistrano)
+Backups are implemented as a ruby service class wrapped in a rake task that's invoked by cron (scheduled via whenever / capistrano). A collection must be listed in the `config_map` of the `/config/deploy/<env>.rb` file to generate a backup.
```

Comment:
This new line isn't right, the backup is powered by the collection lists in `config/collections.yml`